.php_cs.dist

```diff
@@ -1,13 +1,17 @@
 <?php
 
 return PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
     ->setRules([
+        '@PHP56Migration:risky' => true,
+        '@PHPUnit57Migration:risky' => true,
```

Comment:
I've already run the fixes for those two rules in #846 and #847. Adding them to the configuration file ensures any new code will also match those rules.